examples/wifi-echo/server/esp32/main/wifi-echo.cpp

```diff
@@ -421,6 +424,48 @@ std::string createSetupPayload()
     return result;
 };
 
+bool inRfc3986UnreservedChar(const char c)
+{
+    char unreserved[] = { '-', '.', '_', '~' };
```

Comment:
`static const char *` ?

Reply:
Used `static const char []` for better readability